lib/utils.js

```diff
@@ -29,6 +29,9 @@ function configureBinaryLog (opts) {
     let l = this.levels[level];
     if (l < this.levels[this.level]) return; // eslint-disable-line curly
     actualLog(level, prefix, msg);
+
+    // TODO: Callback function to handle logs
+    opts.logHandler && opts.logHandler(level, prefix, msg);
```

Comment:
I would prefer the `if` operator, since it's easier to read

Reply:
also, how about changing the name to `onLogMessage` ?

Reply:
OK, I will update it.
I just make same function name as Appium for ease of use.

Reply:
@mykola-mokhnach Already updated, please check.